book/05-records.html

```diff
@@ -33,11 +33,15 @@ <h1>Records</h1>
     <p>Note that record field names must start with a lowercase
     letter.</p>
 
-    <p>Here's a simple example: a <code>service_info</code> record
-    that represents an entry from the <code>/etc/services</code>
-    file on a typical Unix system. That file is used for keeping
-    track of the well-known port and protocol name for protocols
-    such as ftp or ssh.</p>
+    <p>
+      Here's a simple example: a <code>service_info</code> record that
+      represents an entry from the <code>/etc/services</code> file on
+      a typical Unix system. That file is used for keeping track of
+      the well-known port and protocol name for protocols such as ftp
+      or ssh.  Note that we're going to open <code>Core</code> in this
```

Comment:
I think ftp and ssh are usually capitalised as FTP and SSH, since they are acronyms

Reply:
Will do.